examples/SC_noise_capture.py

```diff
@@ -0,0 +1,96 @@
+import pyspecdata as ps
+import numpy as np
+import os
+from numpy import r_
+import SpinCore_pp as sc
+from datetime import datetime
+
+my_exp_type = "ODNP_NMR_comp/noise_tests"
+assert os.path.exists(ps.getDATADIR(exp_type=my_exp_type))
+config_dict = sc.configuration("active.ini")
```

Comment:
incorporating config into acquisition - note, this is not included in MWE in manuscript